deepinv/datasets/patch_dataset.py

```diff
@@ -0,0 +1,34 @@
+from torch.utils import data
+import torchvision.transforms.v2
+
+class PatchDataset(data.Dataset):
+    r"""
+    Builds the dataset of all patches from a tensor of images.
+
+    :param torch.Tensor imgs: Tensor of images, size: batch size x channels x height x width
+    :param int patch_size: size of patches
+    :param callable or str transforms: data augmentation. None for no augmentation, 'flip and rotate' for including random flips and rotations or callable object.
```

Comment:
I removed this predefined transform entirely and set the default value to None (= no data augmentation). Even though it can give some gain for training a patch normalizing flow, it is not used in the examples and particularly for estimating a GMM one must not use data augmentation (since one has to pass the whole dataset in a single iteration of the EM algorihtm).